### ISSUE TITLE
Builders not working above y=128 in the end

### ISSUE THREAD

**Issue by toddnollan** [BuildCraft/BuildCraft#3262]
Builders, placed in the end, when given a Blueprint that vertically extends above y=128 or placed themselves above y=128 will not build any blocks above that line. They will instead fire the red 'breaking' blocks at the places where blocks are required, as if the spaces were filled.

Found in private pack, tested in both singleplayer and multiplayer, in both creative and survival.
BC version 7.1.14, not listed as fixed in .15 or .16

Confirmed existing in FTB Infinity
Additionally, Fillers seem to think something is there.
Filling each block (by hand) seems to fix the issue for that particular area.

**Comment by TriBlade9** [comment]
Confirmed, in the nether as well.

**Comment by MrGTout** [comment]
This is what happen when I use it on nether even I turn off the excavate.
![2016-04-16_15 15 17](https://cloud.githubusercontent.com/assets/12579912/14583942/6344ad1a-03e6-11e6-8c50-fea7a14583aa.png)
In fact there are nothing.

**Comment by panda2134** [comment]
@MrGTout interesting bug... have you confirmed this with only buildcraft installed?

**Comment by MrGTout** [comment]
Yes

发自我的 iPhone

**Comment by 3TUSK** [comment]
@panda2134 @MrGTout Great...
http://www.mcbbs.net/thread-586890-1-1.html
Another thread that complain about builder not functioning in Nether where y>128. 
> Buildcraft version is unknown...

**Comment by MrDiamond123** [comment]
Is the chunk loaded?
If it is, is it loaded by you or a chunk loader block (What mod is the block from?)?

**Comment by AEnterprise** [comment]
@MrDiamond123 if you can see the chunk then the player is chunkloading it

is this bug still present in the latest 1.8.9 version of BC?

**Comment by AlexIIL** [comment]
Yes. Its caused by
 ````java
    public DimensionProperty(World iWorld, WorldProperty iProp) {
        world = iWorld;
        worldHeight = iWorld.getActualHeight();
        world.addWorldAccess(this);
        worldProperty = iProp;
    }
````
WorldProvider:
````java
    public int getHeight()
    {
        return 256;
    }

    public int getActualHeight()
    {
        return hasNoSky ? 128 : 256;
    }
````

**Comment by AlexIIL** [comment]
The question is whether we want to continue using ````getActualHeight```` or use ````getHeight````. The only place where ````getActualHeight```` is used over ````getHeight```` in vanilla is for portals. (Its actually used all over BC for all patterns and robot bombers)

**Comment by AEnterprise** [comment]
if only portals use it i'd say BC shouldn't be using it, it's a portals only function that has an incorrect naming

**Comment by AEnterprise** [comment]
fixed on 1.8.9 so can be closed
